src/pages/Admin/VenueOwnersModal/VenueOwnerModal.scss

```diff
@@ -1,18 +1,11 @@
 @import "scss/constants";
 
-.venue-owner-modal {
-  .text-input {
-    margin-top: 20px;
-    background: var(--content--under-80a);
-    color: var(--content--over);
+.VenueOwnersModal {
+  width: 100%;
+  padding: $spacing--lg;
 
-    &:-webkit-autofill,
-    &:-webkit-autofill:hover,
-    &:-webkit-autofill:focus,
-    &:-webkit-autofill:active {
-      -webkit-text-fill-color: var(--content--over);
-      box-shadow: box-shadow--input-autofill(var(--content--under-80a)) inset;
-    }
```

Comment:
Are these now global or contained within appropriate inputs?

Reply:
this is added in places where input is used ( admininput, inputfield etc.)